src/datasets/glam.md

```diff
@@ -0,0 +1,115 @@
+# GLAM datasets
+
+## Introduction
+
+The GLAM aggregation tables are useful for getting the SQL view of the data that drives GLAM if more exploration is required. GLAM aims to answer a majority of the "easy" questions of how a probe or metric has changed over time. Exploring the GLAM tables could take that a little farther. If you need to dive deeper or aggregate on a field that isn't included here, consider reading [todo].
```

Comment:
Long paragraphs are a little easier to read / edit if you separate them out per line (I originally found this weird but have become addicted to it). Also I think you missed a reference.

```suggestion
[GLAM](https://glam.telemetry.mozilla.org) aims to answer a majority of the "easy" questions of how a probe or metric has changed over time. 
The GLAM  aggregation tables are useful for accessing the data that drives GLAM if more exploration is required.
Exploring the GLAM tables could take that a little farther, but still has some limitations. 
If you need to dive deeper or aggregate on a field that isn't included here, consider reading [Visualizing Percentiles of a Main Ping Exponential Histogram](https://docs.telemetry.mozilla.org/cookbooks/main_ping_exponential_histograms.html).
```